crates/sui-bridge/src/types.rs

```diff
@@ -250,6 +250,12 @@ pub struct SuiToEthBridgeAction {
     pub sui_bridge_event: EmittedSuiToEthTokenBridgeV1,
 }
 
+impl SuiToEthBridgeAction {
+    pub fn timestamp_seconds(&self) -> Option<u64> {
+        self.sui_bridge_event.timestamp_ms.map(|ms| ms / 1000)
+    }
+}
+
```

Comment:
So this may not work, and we'll likely want a full SuiToEthBridgeActionV2 type since the timestamp can't just be added as its a breaking change to the serialized format that we store in the db :/